### DISCUSSION TITLE
v0.0.1

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/github-action-workflow#2] [Announcements]
## What's Changed
* feat: support github actions uses images by @LinuxSuRen in https://github.com/LinuxSuRen/github-action-workflow/pull/1

## New Contributors
* @LinuxSuRen made their first contribution in https://github.com/LinuxSuRen/github-action-workflow/pull/1

**Full Changelog**: https://github.com/LinuxSuRen/github-action-workflow/commits/v0.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/LinuxSuRen/github-action-workflow/releases/tag/v0.0.1'>v0.0.1</a>.</em>

**Comment by mubarak-j** [comment]
This looks interesting but I don't fully understand how it works. Does it convert the Github workflow into a deployable app via ArgoCD?

**Reply by LinuxSuRen** [reply]
Thanks for your interest in this project. The purpose of this project is to convert the GitHub workflow file to an Argo Workflow. So users don't need to understand Kubernetes resources. And the GitHub workflow is quite popular.

This project also provides an Argo CD CMP (Config Management Plugin). It could convert the GitHub workflow automatically.
